packages/beacon-core/src/transports/clients/CommunicationClient.ts

```diff
@@ -17,6 +18,8 @@ import { clientSessionKeys, serverSessionKeys, SessionKeys } from '@stablelib/x2
 export abstract class CommunicationClient {
   constructor(protected readonly keyPair?: KeyPair) {}
 
+  public events: Map<ClientEvents, Function> = new Map()
```

Comment:
It looks like the `events` map actually holds event handlers, not events alone, but the name indicates otherwise. Maybe we could have a better suited name? 😅

Reply:
I changed the name to "evetnHandlers"